### ISSUE TITLE
Request: refresh / load newest toots when clicking Home

### ISSUE THREAD

**Issue by haayman** [elk-zone/elk#878]
Feature Request: from Twitter I'm used to clicking on the Home link to load all newest tweets/toots. On Elk I used to click on the Elk logo, but since it's mostly hidden now it always takes two clicks now to refresh

**Comment by forestsystem** [comment]
Seconding this! It would be very nice to have this feature, especially since the Elk logo isn't anywhere on the web app for mobile, and swiping downwards doesn't refresh either. The solution I've found is to completely force-quit the web app on mobile and open it again to refresh.

**Comment by ZMYaro** [comment]
I don't think it should automatically reload when you select Home, but I think we should follow the example of other social apps and have an “↑ X new posts” button that stays on the screen no matter how far down you scroll.

Even better would be to also have a “↓ Return” button that would take you back to where you were reading after you read the new posts, but that is probably beyond the scope of this ticket.

**Comment by zeroby0** [comment]
@forestsystem 

> and swiping downwards doesn't refresh either.

Are you on Firefox PWA? Swiping down refreshes posts and tapping Home scrolls to top on Chrome PWA v0.4.1 for me.

**Comment by forestsystem** [comment]
@zeroby0 No, sorry, I should have said! I'm using Safari on an iPhone.

**Comment by giltayar** [comment]
Same problem: Safari PWA has no way to refresh the feed.

**Comment by pacanukeha** [comment]
+1 I find it very strange that there is no in-page action to load new posts and that i have to refresh the whole page. very strange.
